esm-wrapper.mjs

```diff
@@ -0,0 +1,145 @@
+// just pre-load all the stuff that index.js lazily exports
+import {createRequire} from 'module';
+
+const require = createRequire(import.meta.url);
+
+const internalRe = require('./internal/re');
```

Comment:
Please do `import internalRe from './internal/re'` here since this will break bundlers otherwise.